packages/flame/lib/src/events/component_mixins/tap_callbacks.dart

```diff
@@ -16,13 +16,18 @@ mixin TapCallbacks on Component {
   void onTapUp(TapUpEvent event) {}
   void onTapCancel(TapCancelEvent event) {}
 
+  /// The delay (in milliseconds) after which a tap is considered a long tap.
+  int get longTapDelay => 300;
```

Comment:
It's a bit strange that it is coming from this mixin, since it'll only have effect the first time a component is added. I think for this case it might be worth adding a `static` variable to the dispatcher.

Reply:
You mean a mutable static? I don't like the ideia. Is there any alternative?

Reply:
There are few times where static variables are useful, but since passing this value in from the mixin just makes it more confusing for the user since they'll believe that they can set it per tappable component, I think it is a better option.
However, one downside of it is that it does affect the possibility of having multiple different games in the same app with different long tap delays (a very rare use-case). I don't have any other ideas for how to solve this, if you do, feel free to suggest them.

Reply:
@spydon Added an alternative implementation here: https://github.com/flame-engine/flame/pull/3110